lib/export.go

```diff
@@ -41,22 +41,40 @@ func NewExportRequests(node *p2p.QriNode, cli *rpc.Client) *ExportRequests {
 
 // ExportParams defines parameters for the export method
 type ExportParams struct {
-	Ref     repo.DatasetRef
-	RootDir string
-	PeerDir bool
-	Format  string
+	Ref       string
```

Comment:
👌 we should chat at some point about normalizing the name of this field across all lib input structs

Reply:
👍